ackermann_steering_controller/src/ackermann_steering_controller.cpp

```diff
@@ -321,7 +321,13 @@ namespace ackermann_steering_controller{
     }
 
     // Limit velocities and accelerations:
-    const double cmd_dt(period.toSec());
+    const double cmd_dt = (period.toSec() > 0) ? period.toSec() : 0.0;
+    if(cmd_dt < 0)
+    {
+      ROS_ERROR("Invalid time interval, delta time cannot be negative");
+      curr_cmd.lin = 0.0;
+      curr_cmd.ang = 0.0;
+    }
```

Comment:
Your proposed change won't issue the warning, and we shouldn't touch the curr_cmd.

Something like this makes more sense.

```suggestion
    double cmd_dt(period.toSec());
    if (cmd_dt < 0.0)
    {
      ROS_ERROR("Invalid time interval, delta time cannot be negative");
      cmd_dt = 0.0;
    }
```